src/python/pants/backend/python/register.py

```diff
@@ -38,6 +43,10 @@
 from pants.goal.task_registrar import TaskRegistrar as task
 
 
+def global_subsystems():
+  return (SubprocessEnvironment, PythonNativeCode)
```

Comment:
Might want to leave a comment that this is necessary for these subsystems to be recognized as `optionable_rule()`s, maybe (so that others can copy off of this for further v2 work)? This might change in the future if we allow subscoped dependencies.

Reply:
I think it's fine without this comment because it seems to be standard practice to register subsystems this way.

Also, let's say we add a new subsystem that does not at all interact with `optionable_rule()`, then the comment for this function would become misleading.